src/bun.js/bindings/c-bindings.cpp

```diff
@@ -31,13 +31,50 @@ extern "C" void bun_warn_avx_missing(const char* url)
     }
 
     static constexpr const char* str = "warn: CPU lacks AVX support, strange crashes may occur. Reinstall Bun or use *-baseline build:\n  ";
-    const size_t len = strlen(str);
+    static constexpr size_t prefix_len = 98; // Length of str above
```

Comment:
Don't hardcode this value; declare `str` as an array, so you can do `sizeof` on it.